client/src/actions/fetchBusinesses.js

```diff
@@ -58,24 +96,78 @@ export const fetchOneBusinessRequest = id => dispatch =>
  * @param {object} business
  * @returns {object} update business
  */
-export function updateBusiness(business) {
+const updateBusinessSuccess = business => ({
+  type: UPDATE_BUSINESS_SUCCESSFUL,
+  business
+});
+
+/**
+ *
+ * @param {object} error
+ * @returns {object} update error
+ */
+export function updateBusinessfailed(error) {
   return {
-    type: UPDATE_BUSINESS_SUCCESSFUL,
-    business
+    type: UPDATE_BUSINESS_FAILED,
+    error
   };
 }
 
+const updateBusiness = (business, cloudImageUrl) => (
+  dispatch => (
+    axios({
+      method: 'PUT',
+      url: `/api/v1/businesses/${business.id}`,
+      data: {
+        name: business.name,
+        description: business.description,
+        phoneNumber: business.phoneNumber,
+        address: business.address,
+        image: cloudImageUrl,
+        location: business.location,
+        category: business.category,
+        website: business.website
+      }
+    })
+      .then((response) => {
+        dispatch(updateBusinessSuccess(response.data.business));
+      })
+      .catch((error) => {
+        dispatch(updateBusinessfailed(error.response.data.message));
+      }))
+);
```

Comment:
Nice work, great job.

Reply:
Thanks for the feedback, I will appreciate it more if you specify where I did a great job and how I can improve more to continue the great work.